scripts/package-build/build.py

```diff
@@ -91,7 +92,11 @@ def build_package(package: list, patch_dir: Path) -> None:
 
         # Check out the specific commit
         run(['git', 'checkout', package['commit_id']], cwd=repo_dir, check=True)
+    except CalledProcessError as e:
+        print(f"❌ Failed to clone or checkout for package '{repo_name}': {e}")
```

Comment:
```suggestion
        print(f"Failed to clone or checkout for package '{repo_name}': {e}")
```

I'm not a fan of emojis in error messages. At the moment the main reason not to do that is that they aren't anywhere else so they will make messages inconsistent. I think we should discuss the long-term approach together at some point.